timely_beliefs/docs/init.md

```diff
@@ -0,0 +1,91 @@
+    >>> source = tb.BeliefSource("EPEX")
+
+When creating a `BeliefsDataFrame`, a sensor always needs to be passes.
+A source can be passed once for the entire frame, or per belief.
+
+## From a Pandas Series
+
+Create a BeliefsDataFrame from a conventional time series (Pandas Series with DatetimeIndex) together with keyword arguments.
+
+Required arguments:
+
+- either `belief_time` or `belief_horizon`
+- `source` (only a single source can be provided)
+- `sensor`
+
+
+    >>> import pandas as pd
+    >>> s = pd.Series([63, 60], index=pd.date_range(datetime(2000, 1, 3, 9), periods=2, tz=pytz.utc))
+    >>> bdf = tb.BeliefsDataFrame(s, belief_horizon=timedelta(hours=0), source=source, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-04 09:00:00+00:00 0 days         EPEX   0.5                              60
+
+## From a Pandas DataFrame
+
+Pass a Pandas DataFrame with columns ["event_start", "belief_time", "source", "cumulative_probability", "event_value"]. The "cumulative_probability" column is optional (a default of 0.5 will be used if the column is missing).
+
+    >>> df = pd.DataFrame([[63, datetime(2000, 1, 3, 9, tzinfo=pytz.utc), timedelta(hours=0), source], [60, datetime(2000, 1, 3, 10, tzinfo=pytz.utc), timedelta(hours=0), source]], columns=["event_value", "event_start", "belief_horizon", "source"])
+    >>> bdf = tb.BeliefsDataFrame(df, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-03 10:00:00+00:00 0 days         EPEX   0.5                              60
+
+Alternatively, keyword arguments can be used to replace columns that contain unique values for each belief.
+
+    >>> df = pd.DataFrame([[63, datetime(2000, 1, 3, 9, tzinfo=pytz.utc)], [60, datetime(2000, 1, 3, 10, tzinfo=pytz.utc)]], columns=["event_value", "event_start"])
+    >>> bdf = tb.BeliefsDataFrame(df, belief_horizon=timedelta(hours=0), source=source, sensor=sensor)
+    >>> print(bdf)
+                                                                            event_value
+    event_start               belief_horizon source cumulative_probability             
+    2000-01-03 09:00:00+00:00 0 days         EPEX   0.5                              63
+    2000-01-03 10:00:00+00:00 0 days         EPEX   0.5                              60
+
+## From a CSV file
+
+The utility function `tb.read_csv` lets you load a BeliefsDataFrame from a csv file (e.g. example/temperature.csv).
```

Comment:
So `example/temperature.csv` lets me see a working example? Tell me explicitly to go and look there, as this paragraph would not be sufficient for me to implement this way of loading data.